src/common/Dropdown.svelte

```diff
@@ -11,6 +11,7 @@
   export let vertPadding = 10;
   const MENU_OFFSET = 1; // vert offset for menu
 
+  export let name = undefined; // for analytics
```

Comment:
Out of curiosity: Why are you explicitly setting to `undefined` here?

Reply:
I added this prop so I could use it in analytics, but I'm not sure I've tracked down _every_ `<Dropdown>` instance across the site, so this prevents a warning about unused props. I have a check down below so I don't fire events when this is `undefined`.